README.md

```diff
@@ -172,13 +172,22 @@ Function | Armbian | Yocto | Buildroot |
 
 ## 🙌 Contribution
 
-- You don't need to be a programmer to help! [Check out  our list](https://forum.armbian.com/staffapplications/) choose what you wanna do ❤️
-
+#### You don't need to be a programmer to help! 
 - The easiest way to help is by "Starring" our repository - it helps more people find our code.
-
-- You also can maintain and develop [docs](https://github.com/armbian/documentation), [CI](https://github.com/armbian/ci-testing-tools), [autotests](https://github.com/armbian/autotests), [seed torrents](https://forum.armbian.com/topic/4198-seed-our-torrents/), help on [forum moderating](https://forum.armbian.com/topic/12631-help-on-forum-moderating/), [project administration](https://forum.armbian.com/forum/39-armbian-project-administration/), [costs](https://www.armbian.com/donate).
-
-Please make sure to read the [Contributing Guide](.github/CONTRIBUTING.md) before you write any code.
+- [Check out our list of volunteer positions](https://forum.armbian.com/staffapplications/) and choose what you want to do ❤️
+- [Seed torrents](https://forum.armbian.com/topic/4198-seed-our-torrents/)
```

Comment:
@EvilOlaf prepared moving it to docs. If that is ready, change this to docs
https://github.com/armbian/documentation/pull/238/files